api/v2/performanceprofile_validation.go

```diff
@@ -253,3 +255,39 @@ func (r *PerformanceProfile) validateNUMA() field.ErrorList {
 
 	return allErrs
 }
+
+func (r *PerformanceProfile) validateNet() field.ErrorList {
+	var allErrs field.ErrorList
+
+	if r.Spec.Net == nil {
+		return allErrs
+	}
+
+	if *r.Spec.Net.UserLevelNetworking && r.Spec.CPU.Reserved == nil {
```

Comment:
CPU field may be NULL

Reply:
@marcel-apf  if its NULL then we can't set a desired number  to the count no ?  what should be the default behaviour in that case ?  should we just do nothing  ?  maybe issue a warning ?